usdt/src/lib.rs

```diff
@@ -211,6 +212,7 @@
 //! Below is the full list of supported types.
 //!
 //! - `(u?)int(8|16|32|64)_t`
```

Comment:
huh. interesting that we can't do `char` here (see comments in the parser)

Reply:
We could, but I'm not sure how that'd play with handling actual Rust strings. We're kind of reserving `char*` to mean a string, so I'd want to make sure we don't conflate that with a byte array.

Reply:
makes sense..